src/CTA.Rules.Config/Utils.cs

```diff
@@ -191,20 +192,52 @@ public static string ThreadSafeExportStringToFile(string filePath, string conten
             }
         }
 
+
+        public static string CopySolutionToDestinationLocation(string solutionName, string solutionPath)
+        {
+            string slnDirPath = Directory.GetParent(solutionPath).FullName;
+            string root = Path.GetPathRoot(slnDirPath);
+            string relativeSrc = Path.GetRelativePath(root, slnDirPath);
+            
+            var newTempDirPath = Path.Combine(Directory.GetParent(slnDirPath).FullName, Guid.NewGuid().ToString());
```

Comment:
If `slnDirPath` is a root directory (e.g. `C:`), line 202 will throw an error from `GetParent()`. Is this a case we should handle?

Reply:
Not really, sln will always be a level above the base drive. since that is how a project get created when initialized new.